frontend/components/apps/tokens/SecretTokens.tsx

```diff
@@ -590,6 +602,10 @@ export const SecretTokens = (props: { organisationId: string; appId: string }) =
                 {isExpired ? 'Expired' : 'Expires'}{' '}
                 {token.expiresAt ? relativeTimeFromDates(new Date(token.expiresAt)) : 'never'}
               </div>
+
+              {tokenEnvironments.map(({ idx, name }: { idx: number; name: string }) => (
+                <div key={idx}>{name}</div>
+              ))}
```

Comment:
```suggestion
              {tokenEnvironments.map(({ envType }: { envType: string }) => (
                <div key={envType}>{envType}</div>
              ))}
```

* Lets use `envType` instead of `name`, as its shorter
* Try to avoid using indices as list keys where possible